### ISSUE TITLE
Upstream HTML Media Capture to HTML?

### ISSUE THREAD

**Issue by xfq** [w3c/html-media-capture#28]
This specification adds a `capture` attribute to the `input` element. Should we upstream it to the HTML spec?

**Comment by xfq** [comment]
Related: https://github.com/whatwg/html/issues/1102

**Comment by xfq** [comment]
Related: https://github.com/whatwg/html/issues/1102

**Comment by anssiko** [comment]
We've made progress in this space, per [W3C/WHATWG Relationship update](https://www.w3.org/2021/03/WHATWG-W3C-MOU.html):

>WHATWG will integrate the W3C specifications on HTML Media Capture, worklets, resource hints, preload, Media Playback Capture in the HTML and DOM specifications.

Once the relevant bits are integrated, we can finally close this issue. Thanks everyone!

**Comment by plehegar** [comment]
Please, make sure to republish the document as "Discontinued Draft" once the HTML integration is done.

**Comment by xfq** [comment]
The document is currently a REC, and I don't see "Discontinued Draft" in https://w3c.github.io/transitions/nextstep.html?shortname=html-media-capture

**Comment by plehegar** [comment]
@xfq , you're right. it should really be republished as an obsolete or supersede Recommendation.
